src/common/components/cards/cards-visualization-modifier-buttons.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    cardsVisualizationModifiersContainer,
+    expandCollapseAllButton,
+    visualHelperToggle,
+} from 'common/components/cards/cards-visualization-modifiers-buttons.scss';
+import { CardSelectionMessageCreator } from 'common/message-creators/card-selection-message-creator';
+import { NamedFC } from 'common/react/named-fc';
+import { SupportedMouseEvent } from 'common/telemetry-data-factory';
+import { ActionButton, Toggle } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+export type CardsVisualizationModifierButtonsDeps = {
+    cardSelectionMessageCreator: CardSelectionMessageCreator;
+};
+
+export type CardsVisualizationModifierButtonsProps = {
+    deps: CardsVisualizationModifierButtonsDeps;
+    visualHelperEnabled: boolean;
+    allCardsCollapsed: boolean;
+};
+
+export const CardsVisualizationModifierButtons = NamedFC<CardsVisualizationModifierButtonsProps>(
```

Comment:
not thrilled about the name of this component... is this something close to a command bar? The `VisualizationModifier` part of the name kind of doesn't match the expand/collapse button... thoughts?

Reply:
Discussed offline: will leave as is.